src/main/resources/db/changelog/changes/08-create-shopping-carts-table.yaml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-carts-table
+      author: RomanYuzviak
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+                    unique: true
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    references: users(id)
+                    foreignKeyName: user_sk_fk
+                    nullable: false
```

Comment:
If you're going to add soft delete, don't forget about this file

Reply:
agree